atc/db/dbtest/builder.go

```diff
@@ -79,6 +79,7 @@ func (builder Builder) WithWorker(worker atc.Worker) SetupFunc {
 				return fmt.Errorf("team does not exist: %s", worker.Team)
 			}
 
+			//lint:ignore SA4006 linter is wrong
```

Comment:
I think the linter is actually right here. On line 73, we create a new binding for `err` that only exists in this branch, and that's the `err` we replace on this line (while the check outside of this if statement is still checking the one defined on line 71)

Reply:
https://play.golang.org/p/JhkCzCjBfa0

Reply:
Good catch! I totally missed that line all together!